src/main/java/com/couchbase/lite/Attachment.java

```diff
@@ -202,7 +202,7 @@ protected InputStream getBodyIfNew() {
                     BlobStoreWriter writer;
                     try {
                         writer = blobStoreWriterForBody(body, database);
-                    } catch (IOException e) {
+                    } catch (Exception e) {
                         throw new CouchbaseLiteException(e.getMessage(), Status.ATTACHMENT_ERROR);
```

Comment:
Don't we need to treat SymmetricKeyException and IOException as different error? or Is it ok to treat both exception as just attachment error?